### ISSUE TITLE
undefined: BitCurve

### ISSUE THREAD

**Issue by lucas7788** [klaytn/klaytn#197]
Freely describe your questions or issues here.

![image](https://user-images.githubusercontent.com/13885466/64140220-57511c00-ce36-11e9-8119-4d7cc036bbe6.png)

**Comment by ehnuje** [comment]
Thanks for reporting the issue =) However, could you please further describe your issue? e.g., reproduce scenario or system settings. If you report this as a bug report, you can find the form with more details.

**Comment by jeonghonah** [comment]
@lucas7788 This issue might solved by PR #281. If you still have a problem with the latest master branch, please reopen the issue.

**Comment by aidan-kwon** [comment]
@lucas7788 I know it's very late, but I hope this reply help you or other people suffering compiling issue. 

It seems that you were trying to cross-compile Klaytn. Because some packages in Klaytn uses `cgo`, you need to enable cgo and specify a C cross-compiler as follows.
```
set CGO_ENABLED=1
set CC=[c cross compiler]
set GOOS=linux
set GOARCH=amd64
```

If your environment is MAC or Linux, I highly recommend using `Makefile`. 
It will help you to compile or cross-compile Klaytn easily. 
I am sorry for that Klaytn doesn't provide build scripts for Windows environments.
